doc/development/plugins.rst

```diff
@@ -37,21 +37,27 @@ Creating your device
 --------------------
 
 The first step in creating your PennyLane plugin is to create your device class.
-This is as simple as importing the abstract base class :class:`~.QubitDevice` from PennyLane,
+This is as simple as importing the abstract base class :class:`~.Device` from PennyLane,
```

Comment:
Another confusing thing was that we use `QubitDevice` here, although the first few sections very generally refer to the ``Device`` class.

Reply:
😬